components/collector/src/source_collectors/sonarqube/violations.py

```diff
@@ -81,12 +84,15 @@ async def __issue_landing_url(self, issue_key: str) -> URL:
 
     async def _entity(self, issue) -> Entity:
         """Create an entity from an issue."""
+        impacts = [
+            f"{impact['severity']} impact on {impact['softwareQuality']}".lower() for impact in issue.get("impacts", [])
+        ]
         return Entity(
             key=issue["key"],
             url=await self.__issue_landing_url(issue["key"]),
             message=issue["message"],
-            severity=issue.get("severity", "no severity").lower(),
-            type=issue["type"].lower(),
+            impacts=", ".join(impacts),
```

Comment:
Do we test the case where an entity has several impacts? Is that scenario possible (it doesn't seem searchable)?

Reply:
No, that case was missing and yes, it's possible according to the SonarQube docs. I've added an issue with two impacts.